src/runtime-rs/Makefile

```diff
@@ -8,14 +8,252 @@
 # Use the following format:
 # '##VAR VARIABLE_NAME: help about variable'
 # '##TARGET TARGET_NAME: help about target'
-
+PROJECT_TYPE = kata
 PROJECT_NAME = Kata Containers
+PROJECT_TAG = kata-containers
 PROJECT_URL = https://github.com/kata-containers
 PROJECT_COMPONENT = containerd-shim-kata-v2
 CONTAINERD_RUNTIME_NAME = io.containerd.kata.v2
 
+include ../../utils.mk
+
+ARCH_DIR = arch
+ARCH_FILE_SUFFIX = -options.mk
+ARCH_FILE = $(ARCH_DIR)/$(ARCH)$(ARCH_FILE_SUFFIX)
+
+ifeq (,$(realpath $(ARCH_FILE)))
+    $(error "ERROR: invalid architecture: '$(ARCH)'")
```

Comment:
Here, only an `x86_64-options.mk` file is provided, so this should fail in another arch?

Reply:
We only support x86_64 currently, so we assume it will fail for another arch. We will add other options in the future.